packages/gatsby-source-wordpress/src/steps/create-schema-customization/build-types.js

```diff
@@ -126,35 +110,8 @@ const objectType = typeBuilderApi => {
     },
   }
 
-  if (type.interfaces) {
-    objectType.interfaces = type.interfaces
-      .filter(interfaceType => {
-        const interfaceTypeSettings = getTypeSettingsByType(interfaceType)
-
-        return (
-          !interfaceTypeSettings.exclude &&
-          fieldOfTypeWasFetched(type) &&
-          fieldOfTypeWasFetched(interfaceType)
-        )
-      })
-      .map(({ name }) => buildTypeName(name))
-  }
-
-  if (
-    gatsbyNodeTypes.includes(type.name) ||
-    isAGatsbyNode ||
-    // this accounts for Node types that weren't fetched because
-    // they have no root field to fetch a single node of this type
-    // removing them from the schema breaks the build though
-    // @todo instead, if a node type isn't fetched, remove it
-    // from the entire schema
-    type?.interfaces?.find(({ name }) => name === `Node`)
-  ) {
-    // this is used to filter the node interfaces
-    // by different content types (post types)
-    objectType.fields[`nodeType`] = `String`
-
-    objectType.interfaces = [`Node`, ...objectType.interfaces]
+  if (type.interfaces?.includes(`Node`)) {
+    objectType.interfaces = [`Node`]
```

Comment:
Any type that implements node on the origin schema also needs to implement it in Gatsby